data/sles4sap/sap_deployment_automation_framework/SAP_SYSTEM.tfvars

```diff
@@ -854,7 +854,7 @@ use_spn = true
 
 # bom_name is the name of the SAP Application Bill of Materials file
 
-bom_name = "HANA_2_00_075_v0001ms"
+bom_name = "%SDAF_BOM_NAME%"
```

Comment:
This generalizes tfvars template making it usable for multiple kinds of deployments.